src/main/common/time.h

```diff
@@ -39,8 +39,15 @@ typedef uint32_t timeUs_t;
 
 static inline timeDelta_t cmpTimeUs(timeUs_t a, timeUs_t b) { return (timeDelta_t)(a - b); }
 
+typedef enum {
+    DST_EU,
+    DST_USA,
+} tz_dst_country_e;
```

Comment:
Name this `tz_automatic_dst_e` with the following values: `TZ_AUTO_DST_OFF`, `TZ_AUTO_DST_EU`, `TZ_AUTO_DST_USA`. Adjust settings.yaml accordingly.